source/tests/endpoints/plugins/test_requiredInfo.py

```diff
@@ -0,0 +1,33 @@
+import unittest
+
+from endpoints.plugins.business_logic.RequiredInfo import BusinessLogicRequiredInfo
+from model.PluginManager import PluginManager
+from plugins.mock.lock.Lock import Lock
+
+
+class TestEndpointRequiredInfo(unittest.TestCase):
+    """
+    A test for the endpoint RequiredInfo.
```

Comment:
This test does not test the endpoint itself does it it tests the business logic module.